extensions/wikia/AssetsManager/config.php

```diff
@@ -12,7 +12,6 @@
 		'//resources/wikia/libraries/mustache/mustache.js',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "//resources/wikia/libraries/mustache/mustache.js" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)